redhat-marketplace/redhat-marketplace-operator
# Feature/feature/installation gauge

**Work Done**
Created gauges to track installation progress.
* rhm_operator_status_install_succeeded
* rhm_operator_status_install_failed
* rhm_operator_status_install_start_time
* rhm_operator_status_install_end_time
* rhm_operator_status_install_no_secret 

custom metrics are currently exposed on localhost:2112/metrics

**To Test**
Tested locally using minikube
* minikube start
* make setup-minikube
* run the operator locally
* check metrics on localhost:2112/metrics
